builds/any/rootfs/stretch/common/overlay/etc/systemd/system/networking.service.d/switchdev-online.conf

```diff
@@ -1,2 +1,3 @@
 [Unit]
-After=network-pre.target
+After=local-fs.target network-pre.target
+Before=shutdown.target network.target network-online.target
```

Comment:
Maybe add to the commit message, that *ifupdown* [is doing the same](https://salsa.debian.org/debian/ifupdown/-/blob/bd4e78a40d2fb25a6237100a2117e14b39fe6622/debian/networking.service):

```
Requires=ifupdown-pre.service
Wants=network.target
After=local-fs.target network-pre.target apparmor.service systemd-sysctl.service systemd-modules-load.service ifupdown-pre.service
Before=network.target shutdown.target network-online.target
Conflicts=shutdown.target
```

Reply:
Yes, I just tried to follow ifupdown's config to setup this.
Thanks
I will add those commit message next time!